wiki/Tournaments/en.md

```diff
@@ -13,7 +13,7 @@ Official tournaments hosted by the Tournament Management.
 osu! World Cup is a country-based 4v4 Team VS tournament following double-elimination rule. It is an international event and is usually done yearly with the osu!communities around the world working together to make a tournament to create a competitive nature.
 
 | Name | Start | End | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") |
-|------|-------|-----|----------------------------------------------------|------------------------------------------------------|------------------------------------------------------|
+| ------ | ------- | ----- |---------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ |
```

Comment:
Use only three characters per column. Do for every instance.
i.e.
`| --- | --- | --- | --- | --- | --- |`